spec/app/db/repository/authenticator_repository_spec.rb

```diff
@@ -89,6 +89,21 @@
               ::Role['rspec:policy:conjur/authn-oidc/baz-abc123'].destroy
             end
           end
+
+          context 'when webservices status are presents' do
+            before(:each) do
+                ::Resource.create(
+                  resource_id: "rspec:webservice:conjur/authn-oidc/foo-abc123/status",
+                  owner_id: "rspec:policy:conjur/authn-oidc/foo-abc123"
+                )
+              end
+
+            it { expect(repo.find_all(type: 'authn-oidc', account: 'rspec').length).to eq(2) }
```

Comment:
How cumbersome would it be to actually verify the contents of the list, rather than just the count? 

Being explicit about the expected outcome would be a more valuable assurance than just the number of results.

Reply:
would not work since we and not brining over the real resource_id in the repository, but a created one based on the service_id which is the same between the status and normal web-services.